score/scripts/download_cbg.py

```diff
@@ -87,10 +81,7 @@
                         cbg_per_state_list[geoid10_state_id] = []
                     cbg_per_state_list[geoid10_state_id].append(geoid10)
 
-    csv_dir_path = data_path.joinpath(
-        "census",
-        "csv",
-    )
+    csv_dir_path = (data_path / "census" / "csv",)
```

Comment:
I got an error with this line - needed to change to `csv_dir_path = data_path / "census" / "csv"`

Reply:
Fixed